variables.tf

```diff
@@ -24,13 +29,14 @@ variable "versioning_enabled" {
   default     = "true"
 }
 
-variable "mfa_delete_enabled" {
-  description = "If mfa is enabled for bucket deletion"
-  default     = "false"
-}
-
 variable "acl" {
   description = "The access control list assigned to this bucket"
+  default     = "public"
+}
+
+variable "white_list_ip" {
```

Comment:
Can you correct this bit and references to `whitelist` as it's one word. Probably:
```
variable "ip_whitelist" {
    description = "A list of IPs to permit access to the S3 bucket (allows all by default)"
    default     = []
}
```